### PR TITLE
feat: converted everything over to the superior utf8

### PR DESCRIPTION
**Pull request by dustin4242**

### CODE REVIEW

File: Cargo.toml [hunk 1]
```diff
@@ -6,5 +6,5 @@ edition = "2021"
 # See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
 
 [dependencies]
-psf-rs = "1.0.2"
+psf-rs = "2.0.0"
```
Comment (talwat) [inline Cargo.toml:9]: 2.0.2 is available, and it has a few really important fixes.

**Review by talwat — CHANGES_REQUESTED** [review]
The rest looks good! Just the version of psf-rs needs changing.

File: Cargo.toml (hunk 1 above)
Comment (dustin4242) [reply Cargo.toml:9]: understood

File: Cargo.toml [hunk 2]
```diff
@@ -6,5 +6,5 @@ edition = "2021"
 # See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
 
 [dependencies]
-psf-rs = "1.0.2"
+psf-rs = "2.0.2"
```
Comment (dustin4242) [inline Cargo.toml:9]: new version

**Review by talwat — APPROVED** [review]
LGTM, and it seems to work on my machine.

**Pull request merged**